Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -255,6 +255,12 @@ public void Set(MixedRealityRaycastHit hit, RayStep ray, int rayStepIndex, float
             public void Set(RaycastResult result, Vector3 hitPointOnObject, Vector4 hitNormalOnObject, RayStep ray, int rayStepIndex, float rayDistance)
             {
                 raycastHit = default(MixedRealityRaycastHit);
+                raycastHit.point = hitPointOnObject;
+                raycastHit.normal = hitNormalOnObject;
+                raycastHit.distance = rayDistance;
+                raycastHit.transform = result.gameObject.transform;
+                raycastHit.raycastValid = false; //we can't populate all fields of MixedRealityRaycastHit in case of a GraphicsRaycast so we flag the hit data as invalid 
```

Comment:
what are the side effects of not flagging the hit as invalid?

Reply:
i thought it didn't but just saw that this flag is used in the StabilizationPlaneModifier. And actually there's all data available for this raycast to be valid. what's missing is the extra information: texcoords, triangleindex, barycentricCoordinate, lightmapcoord
I will change that flag to true, because those values are actually not used in the entire mrtk code base - i tested by removing them entirely